build-scripts/components/cluster-agent/build.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 
 export INSTALL="${1}/bin"
+export GOEXPERIMENT=opensslcrypto
 mkdir -p "${INSTALL}"
 
-make cluster-agent
+CGO_ENABLED=1 go build -o cluster-agent ./main.go
```

Comment:
This is missing some flags from the makefile, see https://github.com/canonical/microk8s-cluster-agent/blob/main/makefile#L6C2-L6C68

```suggestion
CGO_ENABLED=1 go build -ldflags '-s -w' -o cluster-agent ./main.go
```